app/config_test.go

```diff
@@ -68,12 +69,10 @@ func TestClientConfigWithComputed(t *testing.T) {
 	defer th.TearDown()
 
 	config := th.App.ClientConfigWithComputed()
-	if _, ok := config["NoAccounts"]; !ok {
-		t.Fatal("expected NoAccounts in returned config")
-	}
-	if _, ok := config["MaxPostSize"]; !ok {
-		t.Fatal("expected MaxPostSize in returned config")
-	}
+	_, ok := config["NoAccounts"]
+	require.True(t, ok, "expected NoAccounts in returned config")
+	_, ok = config["MaxPostSize"]
+	require.True(t, ok, "expected MaxPostSize in returned config")
```

Comment:
I would actually suggest using `assert` instead of `require` here. `require` is more for checking setup state for a test (and will fail the test immediately on one not passing) while `assert` is meant to check results (and will let the test run to the end).

Reply:
done